hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/TrashPolicyOzone.java

```diff
@@ -126,9 +126,9 @@ public boolean moveToTrash(Path path) throws IOException {
     Path trashRoot = this.fs.getTrashRoot(path);
 
     String key = path.toUri().getPath();
-    LOG.debug("Key path to moveToTrash: "+ key);
+    LOG.debug("Key path to moveToTrash: {}"+ key);
```

Comment:
this'll still result in an string concantenation.
So change it to:
`LOG.debug("Key path to moveToTrash: {}",key);`